pkg/reconciler/autoscaling/reconciler.go

```diff
@@ -173,9 +173,7 @@ func (c *Base) ReconcileMetric(ctx context.Context, pa *pav1alpha1.PodAutoscaler
 	} else if err != nil {
 		return perrors.Wrap(err, "error fetching metric")
 	} else {
-		// Ignore status when reconciling
-		desiredMetric.Status = metric.Status
-		if !equality.Semantic.DeepEqual(desiredMetric, metric) {
+		if !equality.Semantic.DeepEqual(desiredMetric.Spec, metric.Spec) {
```

Comment:
Will this change impact HPA as well? If so, add unit tests to cover it?

Reply:
There are already tests for metric, I think... but I can take a look.